ui/raidboss/data/05-shb/trial/innocence-ex.js

```diff
@@ -1,8 +1,8 @@
-'use strict';
+﻿'use strict';
```

Comment:
Node doesn't like this line's encoding, Can you try re-pasting it from the original?

Reply:
cactbot files are all utf-8 with no BOM (byte order mark).  You may need to adjust your editor to not add the BOM.  There's a lint error for this because git thinks files with BOM are binary (sometimes?) and it gets frustrating.

See: https://notepadunix2dos.info/removebom.html